### PR TITLE
Bug/content warnings

### PR DESCRIPTION
**Pull request by SrinandhiniM**
Now sensitive contents are blurred and viewed when its is clicked. NSFW contents are hidden and viewd when user clicks the button.

### CODE REVIEW

**Comment by iMagesh** [comment]
@SrinandhiniM please fix the conflicts

**Comment by SrinandhiniM** [comment]
Solved the conflicts

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -76,3 +76,12 @@
...
 </div>
+
+
+
+
+
+
+
+
+
```
Comment (iMagesh) [inline app/views/layouts/_navbar.html.erb:87]: please remove this unwanted space.. it was already removed in other commit but it seem to have appeared again after your conflict. Can you check? @SrinandhiniM

**Comment by SrinandhiniM** [comment]
Extra spaces are now removed and committed now.

**Pull request merged**